frontend/src/Components/Blogs/BlogCard.tsx

```
@@ -0,0 +1,105 @@
+			<div className='p-6'>
+				<div className='flex items-center gap-4 mb-3 flex-wrap'>
+					{blog.tagTitle &&
+						blog.tagTitle.map((tag: string) => (
+							<span
+								key={tag}
+								className='px-2 py-1 bg-accent/20 text-accent rounded-full text-xs font-medium'
+							>
+								{tag}
+							</span>
+						))}
+				</div>
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)